src/app/Components/CommonNavbar.js

```diff
@@ -28,7 +28,7 @@ export default function CommonNavbar(props) {
           <Link className="nav-link" to="/" onClick={() => props.setScrollDestination("whoweare")}>Who We Are</Link>
           <Link className="nav-link" to="/" onClick={() => props.setScrollDestination("whatwedo")}>What We Do</Link>
           <Link className="nav-link" to="/team">Our Team</Link>
-          <Link className="nav-link" to="/" onClick={() => props.setScrollDestination("contactus")}>Contact Us</Link>
+          <Link className="nav-link" to="/contact" onClick={() => props.setScrollDestination("contactus")}>Contact Us</Link>
```

Comment:
Remove the onClick prop; it is for scrolling to the contactus section of the home page, which we are not doing.
```suggestion
          <Link className="nav-link" to="/contact">Contact Us</Link>
```